### PR TITLE
Hindi Translation of Eclipse Collections website #792

### PR DESCRIPTION
**Pull request by nishantraut**
Hindi Translation of Eclipse Collections website #792

### CODE REVIEW

File: site/cn/index.html [hunk 1]
```diff
@@ -111,6 +111,7 @@
                             <li><a href="../ja/index.html">日本語</a></li>
                             <li><a href="../pt-br/index.html">Português-Brasil</a></li>
                             <li><a href="../ru/index.html">Русский</a></li>
+							<li><a href="../hi/index.html">हिंदी</a></li>
```
Comment (nikhilnanivadekar) [inline site/cn/index.html]: fix the indentation. same comment for files of other language

File: site/hi/index.html [hunk 2]
```diff
@@ -0,0 +1,669 @@
+﻿<!DOCTYPE html>
+<!--
+  ~ Copyright (c) 2018 Goldman Sachs and others
```
Comment (nikhilnanivadekar) [inline site/hi/index.html]: Fix the copyright. Please follow the instructions here: https://www.eclipse.org/projects/handbook/#ip-copyright-headers
Refer to the comments about "owner"

**Comment by nikhilnanivadekar** [comment]
I have started reviewing this PR. However, I will be traveling today. I will try to get back with all the comments by tomorrow.
Thanks for your contribution.
@cguntur-bnym if you would like to review as well as you travel, you can help us out. Thank you!

**Comment by nishantraut** [comment]
Thanks please review main file as well, i plan to fix all on weekend.

File: site/hi/index.html [hunk 3]
```diff
@@ -0,0 +1,669 @@
...
+                </div>
+                <div class="col-lg-4 col-sm-4">
+                    <h3>सामुदायिक लिंक</h3>
+                        <p class="regular">एक्लिप्स पीएमआई: <a href="https://projects.eclipse.org/projects/technology.collections">एक्लिप्स परियोजना प्रबंधन आधारिक संरचना</a></p>
+                        <p class="regular">गिटहब: <a href="https://github.com/eclipse/eclipse-collections">एक्लिप्स-कलेक्शन</a></p>
+                        <p class="regular">मुद्दे: <a href="https://github.com/eclipse/eclipse-collections/issues">गिटहब मुद्दे</a></p>
+                        <p class="regular">ईमेल की सूची: <a href="https://dev.eclipse.org/mailman/listinfo/collections-dev">कलेक्शन-देव</a> मेलिंग सूची</p>
+                        <p class="regular">विकी: <a href="https://github.com/eclipse/eclipse-collections/wiki">गिटहब विकि</a></p>
+                        <p class="regular">मंच: <a href="https://www.eclipse.org/forums/index.php?t=thread&amp;frm_id=329">एक्लिप्स कलेक्शन मंच</a></p>
+                        <p class="regular">स्टाकॉवेर्फ्लो: टैग के साथ <a href="http://stackoverflow.com/questions/tagged/eclipse-collections">एक्लिप्स-कलेक्शन</a></p>
```
Comment (nikhilnanivadekar) [inline site/hi/index.html]: I think stackoverflow has a typo in it.

File: site/hi/index.html [hunk 4]
```diff
@@ -0,0 +1,669 @@
...
+    <!-- /.banner -->
+
+    <!-- Footer -->
+    <footer>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-12">
+                    <ul class="list-inline">
+                        <li>
+                            <p class="copyright text-muted small">कॉपीराइट &copy; 2018 एक्लिप्स फाउंडेशन. सर्वाधिकार सुरक्षित. </p>
```
Comment (nikhilnanivadekar) [inline site/hi/index.html]: keep the copyright in English

File: site/hi/index.html [hunk 5]
```diff
@@ -0,0 +1,669 @@
...
+                        <li>
+                            <a class="small" href="http://www.eclipse.org">एक्लिप्स फाउंडेशन</a>
+                        </li>
+                        <li class="footer-menu-divider">&sdot;</li>
+                        <li>
+                            <a class="small" href="http://www.eclipse.org/legal">कानूनी</a>
+                        </li>
+                        <li class="footer-menu-divider">&sdot;</li>
+                        <li>
+                            <a class="small" href="http://www.eclipse.org/legal/privacy.php">गोपनीयता</a>
```
Comment (nikhilnanivadekar) [inline site/hi/index.html]: this means confidential. I dont quite know what it is for privacy.

**Comment by nikhilnanivadekar** [comment]
Submitted my review. @cguntur-bnym can you also take a look please.

**Comment by cguntur-bnym** [comment]
I could not directly review from git diffs. I checked out the code from @nishantraut 's repo. Have some comments about language, typos and punctuations. I'll start typing up the english letters so Nishant can convert to Hindi letters. Punctuations in Hindi need to be followed. Poorna viram (|) is the right way to end a sentence, not a period (.). Some sentences don't read correctly at the moment, will discuss with Nishant as well. 

Brings back good old memories though !

Some examples: BiMap is pronounced as "Bai Map", but the Hindi text shows as "Bee Map". The Multimap has a typo (reads as Multipap) etc.

The first para does not read correct to me at all :-(

File: site/hi/index.html [hunk 6]
```diff
@@ -0,0 +1,669 @@
...
+    <a class="anchor" name="start"></a>
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">से शुरू करें <br>Eclipse Collections</h2>
+                    <p class="lead">Eclipse Collections अब तक का सबसे अच्छा जावा संग्रह ढांचा है
```
[the remark this answers is not in the document]
Comment (cguntur-bnym) [reply site/hi/index.html]: sarvashreshta ? Avval is mumbayya ...

File: site/hi/index.html [hunk 7]
```diff
@@ -0,0 +1,669 @@
...
+						यह आपके जावा विकास में खुशी लाता है.
+                        <br><br>अपने पसंदीदा बिल्ड टूल के साथ इसे आज अपने कोडबेस में एकीकृत करें!</p>
```
[the remark this answers is not in the document]
Comment (cguntur-bnym) [reply site/hi/index.html]: ekikrut would be the right usage here.

File: site/hi/index.html [hunk 8]
```diff
@@ -0,0 +1,669 @@
...
+                <div class="col-lg-7 col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#mutable" data-toggle="tab">परस्पर कलेक्शन</a></li>
+                        <li><a href="#immutable" data-toggle="tab">अपरिवर्तनीय कलेक्शन</a></li>
+                        <li><a href="#primitives" data-toggle="tab">आदिम कलेक्शन</a></li>
+                    </ul>
+                    <div id="container-types" class="tab-content">
+                        <div class="tab-pane fade in active" id="mutable">
+                            <pre  class="prettyprint lang-java">
+//अस्थायी लिस्ट का आरम्भ empty(), of(), with() विधि के साथ
```
[the remark this answers is not in the document]
Comment (cguntur-bnym) [reply site/hi/index.html]: parivartansheel ?

File: site/hi/index.html [hunk 9]
```diff
@@ -0,0 +1,669 @@
...
+ImmutableMap&lt;String, String&gt; immutableMap =
+  Maps.immutable.with("key1", "value1", "key2", "value2", "key3", "value3");
+ImmutableMultimap&lt;String, String&gt; immutableMultimapWithList =
+  Multimaps.immutable.list.with("key1", "value1-1", "key1", "value1-2", "key2","value2-1");
+ImmutableBiMap&lt;String, String&gt; immutableBiMap =
+  BiMaps.immutable.with("key1", "value1", "key2", "value2", "key3", "value3");</pre>
+                        </div>
+                        <div class="tab-pane fade" id="primitives">
+                            <pre  class="prettyprint lang-java">
+//सभी 8 आदिम प्रकारों के लिए अस्थायी और स्थायी लिस्ट, सेट, बैग, स्टैक और मैप्स उपलब्ध हैं
```
[the remark this answers is not in the document]
Comment (cguntur-bnym) [reply site/hi/index.html]: Agree with Nikhil. Aadim is a literal translation of Primitive. It means raw or premature for most linguistic purposes.

File: site/hi/index.html [hunk 10]
```diff
@@ -0,0 +1,669 @@
...
+    <a class="anchor" name="memory"></a>
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-6 col-lg-6 col-sm-push-6 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">मेमोरी कुशल कंटेनरों के साथ <br>स्मृति पदचिह्न कम करें</h2>
```
[the remark this answers is not in the document]
Comment (cguntur-bnym) [reply site/hi/index.html]: literally type "memory footprint" in Hindi is my suggestion.

File: site/hi/index.html [hunk 11]
```diff
@@ -0,0 +1,669 @@
...
+    <a class="anchor" name="learn"></a>
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">कटा के साथ जानें एक्लिप्स कलेक्शन</h2>
```
[the remark this answers is not in the document]
Comment (cguntur-bnym) [reply site/hi/index.html]: "Kaata"

File: site/hi/index.html [hunk 12]
```diff
@@ -0,0 +1,669 @@
...
+                <a href="https://github.com/eclipse/eclipse-collections/wiki/Conference-talks-and-meetups">सम्मेलनों</a>
+		2012 में JVM समिट और 2014 JavaOne में शामिल था.
+                A performance comparison between the parallel lazy implementations of Java 8, Scala and GS Collections was presented at
+		जावा 8, स्काला और जीएस कलेक्शन के बीच समानांतर आलसी कार्यान्वयन एक प्रदर्शन की तुलना पर प्रस्तुत किया गया
+                <a href="http://www.infoq.com/presentations/java-streams-scala-parallel-collections">QCon New York</a> 2014 में.
+                Also <a href="https://github.com/eclipse/eclipse-collections/wiki/Articles">लेख</a> GS Collections के बारे में
+                (<a href="http://www.infoq.com/articles/GS-Collections-by-Example-1">भाग 1</a> /
+                <a href="http://www.infoq.com/articles/GS-Collections-by-Example-2">भाग 2</a>)
+                InfoQ.com पर प्रकाशित किया गया
+                         जीएस कलेक्शन के रचनाकारों के उदाहरणों और साक्षात्कारों के माध्यम से इस पुस्तकालय की क्षमताओं को दिखाया.
```
[the remark this answers is not in the document]
Comment (cguntur-bnym) [reply site/hi/index.html]: type "laibrary" in Hindi ?

File: site/hi/index.html (hunk 3 above)
Comment (cguntur-bnym) [reply site/hi/index.html]: it reads as "stalk overflow". Use the double-aiy matra

File: site/hi/index.html (hunk 5 above)
Comment (cguntur-bnym) [reply site/hi/index.html]: literally type "privacy" in Hindi here. While gopneeyatha means privacy, the translation does not work.

File: site/hi/index.html (hunk 2 above)
Comment (nishantraut) [reply site/hi/index.html]: fixed

File: site/cn/index.html (hunk 1 above)
Comment (nishantraut) [reply site/cn/index.html]: fixed

File: site/hi/index.html (hunk 2 above)
Comment (nishantraut) [reply site/hi/index.html]: just changed year to 2020 as not understood link.
Is whole copyright should be in new format as below ?

/********************************************************************************
 * Copyright (c) {year} {owner}[ and others] 1
 *
 * This program and the accompanying materials are made available under the 2
 * terms of the Eclipse Public License 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 *
 * SPDX-License-Identifier: EPL-2.0 3
 *
 * Contributors: 4
 *   {name} - initial API and implementation
 ********************************************************************************/

File: site/hi/index.html [hunk 13]
```diff
@@ -0,0 +1,669 @@
...
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="shortcut icon" type="image/x-icon" href="../favicon.ico" />
+
+    <title>Eclipse Collections - सुविधाएँ जो आपको चाहिए संग्रह के साथ. </title>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: removed

[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: kept Aapki chahiti suvidhaye aapki jarurat ke Collections ke saath

File: site/hi/index.html [hunk 14]
```diff
@@ -0,0 +1,669 @@
...
+    <!-- Header -->
+    <a class="anchor" name="about"></a>
+    <div class="intro-header">
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-2 col-lg-8">
+                    <div class="intro-message">
+                        <h1>Eclipse <span style="font-weight: 400">Collections</span></h1>
+                        <h3>- सुविधाएँ जो आपको चाहिए संग्रह के साथ -</h3>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: updated

File: site/hi/index.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: Modified as "Eclipse Collections ab tak ka sabse sarvashreshta Java collections sangraha hai jo aapke Java programming mein anand lata hai."

File: site/hi/index.html [hunk 15]
```diff
@@ -0,0 +1,669 @@
...
+        </div>
+    </div>
+    <!-- History  -->
+
+    <a class="anchor" name="refGuide"></a>
+    <div class="content-section-a">
+        <div class="container-fluid">
+            <h2 class="section-heading">निर्देशिका</h2>
+            <p class="lead">
+                          पुस्तकालय का एक व्यापक संदर्भ गाइड GitHub पर उपलब्ध है:
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html [hunk 16]
```diff
@@ -0,0 +1,669 @@
...
+            <div class="row">
+                <div class="col-lg-4 col-sm-4">
+                    <h3>योगदान</h3>
+                    <p class="regular">
+                                      हम योगदान का स्वागत करते हैं!
+                        <br><br>
+			एक्लिप्स कलेक्शन प्रोजेक्ट रोडमैप मिल सकता है <a href="https://github.com/eclipse/eclipse-collections/wiki/Roadmap">यहाँ</a>.
+                        <br><br>
+                                      हम GitHub में पुल रिक्वेस्ट के माध्यम से योगदान स्वीकार करते हैं।
+			कृपया देखें <a href="https://github.com/eclipse/eclipse-collections/blob/master/CONTRIBUTING.md">कैसे योगदान</a>करनेका आरंभ करें.
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: updated

File: site/hi/index.html (hunk 3 above)
Comment (nishantraut) [reply site/hi/index.html]: fixed

File: site/hi/index.html (hunk 4 above)
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html (hunk 5 above)
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html [hunk 17]
```diff
@@ -0,0 +1,669 @@
...
+
+    <a class="anchor" name="start"></a>
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">से शुरू करें <br>Eclipse Collections</h2>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: updated

**Comment by nishantraut** [comment]
> I could not directly review from git diffs. I checked out the code from @nishantraut 's repo. Have some comments about language, typos and punctuations. I'll start typing up the english letters so Nishant can convert to Hindi letters. Punctuations in Hindi need to be followed. Poorna viram (|) is the right way to end a sentence, not a period (.). Some sentences don't read correctly at the moment, will discuss with Nishant as well.
> 
> Brings back good old memories though !
> 
> Some examples: BiMap is pronounced as "Bai Map", but the Hindi text shows as "Bee Map". The Multimap has a typo (reads as Multipap) etc.
> 
> The first para does not read correct to me at all :-(

As requested ended sentences with poorna viram(|) instead of period(.). Fixed bimap and multimap. Please let me know if there is any changes required :)

File: site/hi/index.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: kept ekikrut

File: site/hi/index.html [hunk 18]
```diff
@@ -0,0 +1,669 @@
...
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-7 col-lg-5 col-sm-push-6  col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">समृद्ध, संक्षिप्त और पढ़ने योग्य एपीआई</h2>
+                    <p class="lead">जावा 8 लैम्ब्डा के भाव और विधि संदर्भों की शक्ति को अधिकतम करें
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html [hunk 19]
```diff
@@ -0,0 +1,669 @@
...
+						समृद्ध एपीआई सीधे आपके संग्रह पर उपलब्ध हैं.
+                    </p>
+                </div>
+                <div class="col-lg-pull-5 col-lg-7 col-sm-pull-6  col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#method" data-toggle="tab">Eclipse Collections विधि रेफरी के साथ</a></li>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html [hunk 20]
```diff
@@ -0,0 +1,669 @@
...
+
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">कई कंटेनर प्रकार जिनमें अपरिवर्तनीय कलेक्शन, आदिम कलेक्शन, बिमैप, मल्टीपैप और बैग शामिल हैं</h2>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: fixed

File: site/hi/index.html [hunk 21]
```diff
@@ -0,0 +1,669 @@
...
+                    <p class="lead">आप विभिन्न कंटेनर प्रकारों का उपयोग कर सकते हैं, तात्कालिकता के लिए आसान फैक्ट्री के तरीकों का लाभ उठा सकते हैं.</p>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html [hunk 22]
```diff
@@ -0,0 +1,669 @@
...
+                </div>
+                <div class="col-lg-7 col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#mutable" data-toggle="tab">परस्पर कलेक्शन</a></li>
+                        <li><a href="#immutable" data-toggle="tab">अपरिवर्तनीय कलेक्शन</a></li>
+                        <li><a href="#primitives" data-toggle="tab">आदिम कलेक्शन</a></li>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: kept parivartansheel

File: site/hi/index.html [hunk 23]
```diff
@@ -0,0 +1,669 @@
...
+MutableMap&lt;String, String&gt; mutableMap =
+  Maps.mutable.with("key1", "value1", "key2", "value2", "key3", "value3");
+MutableMultimap&lt;String, String&gt; multimapWithList =
+  Multimaps.mutable.list.with("key1", "value1-1", "key1", "value1-2", "key2","value2-1");
+MutableBiMap&lt;String, String&gt; mutableBiMap =
+  BiMaps.mutable.with("key1", "value1", "key2", "value2", "key3", "value3");</pre>
+                        </div>
+                        <div class="tab-pane fade" id="immutable">
+                            <pre  class="prettyprint lang-java">
+//स्थायी लिस्ट का आरम्भ empty(), of(), with() विधि के साथ
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html [hunk 24]
```diff
@@ -0,0 +1,669 @@
...
+ImmutableList&lt;String&gt; immutableListEmpty =
+  Lists.immutable.empty();
+ImmutableList&lt;String&gt; immutableListOf =
+  Lists.immutable.of("One", "One", "Two", "Three");
+ImmutableList&lt;String&gt; immutableListWith =
+  Lists.immutable.with("One", "One", "Two", "Three");
+
+//विभिन्न कंटेनर प्रकार उपलब्ध हैं
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: kept primitive

File: site/hi/index.html (hunk 10 above)
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: updated to memory kushal container ke saath memory ka istamaal dakshatase kare

File: site/hi/index.html [hunk 25]
```diff
@@ -0,0 +1,669 @@
...
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-6 col-lg-6 col-sm-push-6 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">मेमोरी कुशल कंटेनरों के साथ <br>स्मृति पदचिह्न कम करें</h2>
+                    <p class="lead">Eclipse Collections सेट्स और मैप्स के साथ-साथ आदिम कलेक्शन के स्मृति कुशल कार्यान्वयन प्रदान करता है. </p>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html [hunk 26]
```diff
@@ -0,0 +1,669 @@
...
+                </div>
+                <div class="col-lg-pull-6 col-lg-6 col-sm-pull-6 col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#set" data-toggle="tab">सेट</a></li>
+                        <li><a href="#map" data-toggle="tab">मैप्स</a></li>
+                        <li><a href="#ints" data-toggle="tab">आदिम कलेक्शन</a></li>
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html (hunk 11 above)
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: fixed :)

File: site/hi/index.html [hunk 27]
```diff
@@ -0,0 +1,669 @@
...
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">कटा के साथ जानें एक्लिप्स कलेक्शन</h2>
+                    <p class="lead">
+			सवारी लो <a href="https://github.com/eclipse/eclipse-collections-kata">एक्लिप्स कलेक्शन कटा</a>, एक्लिप्स कलेक्शन की दुनिया की खोज करने के लिए एक सरल और मजेदार तरीका है.
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: fixed

File: site/hi/index.html [hunk 28]
```diff
@@ -0,0 +1,669 @@
...
+                        <li><a href="#solution" data-toggle="tab">नमूना सुलझाव</a></li>
+                    </ul>
+                    <div class="tab-content">
+                        <div class="tab-pane fade in active" id="exercise">
+                            <pre  class="prettyprint lang-java">
+@Test
+public void getFirstNamesOfAllPeople()
+{
+  MutableList&lt;Person&gt; people = this.people;
+  MutableList&lt;String&gt; firstNames = null; //Replace null, with a transformation method on MutableList.
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: translated as above

File: site/hi/index.html [hunk 29]
```diff
@@ -0,0 +1,669 @@
...
+    <!-- Kata -->
+
+    <a class="anchor" name="history"></a>
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <h2 class="section-heading">एक्लिप्स कलेक्शन का इतिहास</h2>
+            <p class="lead">
+		एक्लिप्स कलेक्शंस की उत्पत्ति 2004 में गोल्डमैन सच्स में कारमेल नामक कलेक्शन फ्रेमवर्क के रूप में शुरू हुई थी.
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: kept it in english only

File: site/hi/index.html [hunk 30]
```diff
@@ -0,0 +1,669 @@
...
+				
+		तब से फ्रेमवर्क विकसित हुआ है, और 2012 में, यह एक परियोजना के रूप में गिटहब के लिए खुला है
+                <a href="https://github.com/goldmansachs/gs-collections">जीएस कलेक्शन</a>.
+                <br><br>
+		जीएस कलेक्शंस कई बार प्रस्तुत किया गया है
+                <a href="https://github.com/eclipse/eclipse-collections/wiki/Conference-talks-and-meetups">सम्मेलनों</a>
+		2012 में JVM समिट और 2014 JavaOne में शामिल था.
+                A performance comparison between the parallel lazy implementations of Java 8, Scala and GS Collections was presented at
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: deleted

File: site/hi/index.html (hunk 12 above)
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: updated to laibrary

File: site/hi/index.html [hunk 31]
```diff
@@ -0,0 +1,669 @@
...
+                (<a href="http://www.infoq.com/articles/GS-Collections-by-Example-1">भाग 1</a> /
+                <a href="http://www.infoq.com/articles/GS-Collections-by-Example-2">भाग 2</a>)
+                InfoQ.com पर प्रकाशित किया गया
+                         जीएस कलेक्शन के रचनाकारों के उदाहरणों और साक्षात्कारों के माध्यम से इस पुस्तकालय की क्षमताओं को दिखाया.
+                <br><br>
+
+                          कई वर्षों से, एक ही कंपनी के लगभग 40 या तो डेवलपर्स ने संग्रह ढांचे में योगदान दिया है।.
+                <br><br>
+
+                          ओपन सोर्स प्रोजेक्ट की सर्वश्रेष्ठ प्रकृति को अधिकतम करने के लिए, जीएस कलेक्शंस को एक्लिप्स फाउंडेशन में स्थानांतरित कर दिया गया है, 2015 में एक्लिप्स कलेक्शंस के रूप में फिर से ब्रांडेड.
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: modified please check

**Comment by nishantraut** [comment]
Thanks for the review @nikhilnanivadekar and @cguntur-bnym . All comments are fixed, please check once and let me know if there are any changes required. Thanks once again for opportunity.

File: site/hi/index.html [hunk 32]
```diff
@@ -0,0 +1,669 @@
...
+                            <pre  class="prettyprint lang-java">
+//अस्थायी लिस्ट का आरम्भ empty(), of(), with() विधि के साथ
+MutableList&lt;String&gt; mutableListEmpty =
+  Lists.mutable.empty();
+MutableList&lt;String&gt; mutableListOf =
+  Lists.mutable.of("One", "One", "Two", "Three");
+MutableList&lt;String&gt; mutableListWith =
+  Lists.mutable.with("One", "One", "Two", "Three");
+
+//विभिन्न कंटेनर प्रकार उपलब्ध हैं
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: done

File: site/hi/index.html [hunk 33]
```diff
@@ -0,0 +1,669 @@
...
+		जावा 8, स्काला और जीएस कलेक्शन के बीच समानांतर आलसी कार्यान्वयन एक प्रदर्शन की तुलना पर प्रस्तुत किया गया
+                <a href="http://www.infoq.com/presentations/java-streams-scala-parallel-collections">QCon New York</a> 2014 में.
+                Also <a href="https://github.com/eclipse/eclipse-collections/wiki/Articles">लेख</a> GS Collections के बारे में
+                (<a href="http://www.infoq.com/articles/GS-Collections-by-Example-1">भाग 1</a> /
+                <a href="http://www.infoq.com/articles/GS-Collections-by-Example-2">भाग 2</a>)
+                InfoQ.com पर प्रकाशित किया गया
+                         जीएस कलेक्शन के रचनाकारों के उदाहरणों और साक्षात्कारों के माध्यम से इस पुस्तकालय की क्षमताओं को दिखाया.
+                <br><br>
+
+                          कई वर्षों से, एक ही कंपनी के लगभग 40 या तो डेवलपर्स ने संग्रह ढांचे में योगदान दिया है।.
```
[the remark this answers is not in the document]
Comment (nishantraut) [reply site/hi/index.html]: kept as framework

**Comment by nikhilnanivadekar** [comment]
@nishantraut it might be easier to review if you can squash the commits.
@cguntur-bnym can you please review?

**Comment by nishantraut** [comment]
> @nishantraut it might be easier to review if you can squash the commits.
> @cguntur-bnym can you please review?

@nikhilnanivadekar commits are squashed now, @cguntur-bnym please review now.

**Comment by nishantraut** [comment]
नमस्ते चंद्रा @cguntur-bnym  और निखिल @nikhilnanivadekar 
कृपया इस पुल रिक्वेस्ट को रिव्यु करके मर्ज कीजिये। 
धन्यवाद्।

**Comment by donraab** [comment]
@cguntur-bnym @nikhilnanivadekar Any update here?  Is this ready to merge?

**Comment by nikhilnanivadekar** [comment]
@cguntur-bnym needs to review and approve this. Don’t merge till he approves .

**Comment by cguntur-bnym** [comment]
Picking this up today

**Comment by cguntur-bnym** [comment]
काटा के साथ जानें एक्लिप्स कलेक्शन - Seekhein not Jaaney

इस विशेष काटा को यूनिट परीक्षणों की एक श्रृंखला के रूप में स्थापित किया जाता है जो विफल हो जाते हैं - JO vophal ho jaatey hain does not communicate the intent corrently. The right language should mean "meant to fail" (जान भुज कर विफल हैं)? 

कारमेल - CarAmel not Carmel (कैरमेल?)

एक ही कंपनी के लगभग 40 या तो डेवलपर्स ने संग्रह फ्रेमवर्क में योगदान दिया है - why या तो?

कई वर्षों से, एक ही कंपनी के लगभग 40 या तो डेवलपर्स ने संग्रह फ्रेमवर्क में योगदान दिया है। (why present tense?)

... एक्लिप्स फाउंडेशन में स्थानांतरित कर दिया गया है ... (Why present tense?)

Than you for incorporating the other changes, @nishantraut !

**Comment by nishantraut** [comment]
> काटा के साथ **जानें** एक्लिप्स कलेक्शन - Seekhein not Jaaney
> 
> इस विशेष काटा को यूनिट परीक्षणों की एक श्रृंखला के रूप में स्थापित किया जाता है जो **विफल हो जाते हैं** - **_Jo viphal ho jaatey hain_** does not communicate the intent corrently. The right language should mean "meant to fail" (_जान भुज कर विफल हैं_)?
> 
> **कारमेल** - CarAmel not Carmel (कैरमेल?)
> 
> एक ही कंपनी के लगभग 40 **या तो** डेवलपर्स ने संग्रह फ्रेमवर्क में योगदान दिया है - why या तो?
> 
> कई वर्षों से, एक ही कंपनी के लगभग 40 या तो डेवलपर्स ने संग्रह फ्रेमवर्क में योगदान दिया **है**। (why present tense?)
> 
> ... एक्लिप्स फाउंडेशन में स्थानांतरित कर दिया गया **है** ... (Why present tense?)
> 
> Than you for incorporating the other changes, @nishantraut !

Thanks @cguntur-bnym for review, above reported issues are fixed, @nikhilnanivadekar if it is looking good can you please merge.

**Comment by nishantraut** [comment]
Hi @nikhilnanivadekar I have fixed all the comments, please let me know if anything is needed from my side to merge this PR.

**Comment by nikhilnanivadekar** [comment]
It needs @cguntur-bnym to approve. I’ll take a look later today for any other housekeeping issues and let you know. Thanks for your patience and contribution

**Comment by cguntur-bnym** [comment]
LGTM !

**Comment by nishantraut** [comment]
Hi @nikhilnanivadekar, @cguntur-bnym has given go ahead. Please let me know if anything is needed from my side to merge this PR.

**Comment by nishantraut** [comment]
Hi @nikhilnanivadekar Thanks for the feedback, all three points are addressed.

**Comment by nikhilnanivadekar** [comment]
This looks good. Sorry to be a pain, but regarding the commit message I meant this:
“Add Hindi Translation of Eclipse Collections website. Fixes #792”

**Comment by nikhilnanivadekar** [comment]
@nishantraut any updates on the amend of git commit message? Also can you please rebase on to upstream master. Thank you!

**Comment by nishantraut** [comment]
Hi @nikhilnanivadekar Amended commit message and rebase onto upstream master.

**Comment by nikhilnanivadekar** [comment]
Thanks a ton for your contribution!

**Comment by nikhilnanivadekar** [comment]
The website is published: https://www.eclipse.org/collections/hi/index.html

**Pull request merged**